modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1983,6 +1983,57 @@ TEST_P(Test_ONNX_nets, Googlenet)
     expectNoFallbacksFromIE(net);
 }
 
+TEST_P(Test_ONNX_nets, ResNet50)
```

Comment:
TODO: remove this test before merge. It is for speed and accuracy testing currently.

Reply:
Removed.